Engine/src/Engine/EngineUI/GamePanel.h

```diff
@@ -8,7 +8,8 @@ namespace Engine
 	{
 	public:
 		void setInitialPosition();
-		void show(const Window& window);
+		void show(Window& window);
+		glm::vec2 getPos();
```

Comment:
can we change this method name to getPosition to be consistent with the other methods?

Reply:
Renamed